e2e/cypress/integration/markdown/markdown_image_spec.js

```diff
@@ -72,8 +72,10 @@ describe('Markdown', () => {
         cy.getLastPostId().then((postId) => {
             // # Get the image and simulate a click.
             cy.get(`#postMessageText_${postId}`).find('img.markdown-inline-img').
-                should('have.css', 'height', '143px').
-                and('have.css', 'width', '894px').
+                and((inlineImg) => {
+                    expect(inlineImg.height()).to.be.closeTo(143, 2);
+                    expect(inlineImg.width()).to.be.closeTo(894, 2);
+                }).
```

Comment:
For me the tests fails because of the following reasons:

1. markdown_image_spec.js

- with in-line images 1
expected <div.style--none> to have text **Mattermost/platform build status: **, but the text was Mattermost/platform build status: Build Status

- with in-line images 2
expected '<img.markdown-inline-img.markdown-inline-img--hover.cursor--pointer.a11y--active>' to have attribute 'src' with the value 'http://localhost:8065/api/v4/image?url=https%3A%2F%2Fgithub.githubassets.com%2Ffavicon.ico', but the value was 'https://github.githubassets.com/favicon.ico'

- opens image preview window when image is clicked
expected 151.672 to be close to 143 +/- 2

Reply:
- with in-line images 1 > `Build Status` should've not included as it's an `alt` of `img`.  I relaxed the test by using `contain`
- it's due to your image proxy setting.  I added config update.
- maybe the image is still in the process of transforming the size. I added extra check to buy some time in rendering children components.

Reply:
PTAL if it works for you, thanks

Reply:
Thanks @saturninoabril. Let me check